system/edit-config

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env sh
```

Comment:
`sh` and `bash` are not the same in all systems, on Slackware, for example, we have this:

```sh
bash-5.1$ /usr/bin/env sh
sh-5.1$ 
exit
bash-5.1$ /usr/bin/env bash
bash-5.1$ 
exit
bash-5.1$ 
```

Should we use `bash` instead `sh` here?

Reply:
The choice to use plain `sh` here is intentional, as we really need this to work on systems where we don’t have `bash` (that, and the original script also used plain `sh` instead of `bash`).